browser/about_flags.cc

```diff
@@ -362,9 +362,18 @@
       kOsAndroid,                                                             \
       FEATURE_VALUE_TYPE(safe_browsing::features::kBraveAndroidSafeBrowsing), \
   })
+#define BRAVE_ZERO_DAY_FLAG_ANDROID                                        \
+  EXPAND_FEATURE_ENTRIES({                                                 \
+      "brave-zero-day-flag-android",                                       \
+      "ZeroDayFlag flag for product test",                                 \
+      "This flag will be set through griffin to perform product testing",  \
```

Comment:
just ouf of curiousity, what sort of features would it cover? I.e. how is it different than normal per-feature flags ?

Reply:
@iefremov it would be mainly used for product A/B testing starting with new onboarding on android.

Reply:
@deeppandya I understand, but any feature can be used for A/B testing, i.e. `kPlaylist` or something. I'm trying to understand an exact scenario of using of this flag (wondering b/c we try to keep an eye on what's changed via griffin for performance sheriffing purposes

Reply:
Sorry @iefremov for late response. i missed the comment here. Currently we don't have any feature associated with it. @timchilds can provide more details on why we need to add the flag.